pyproject.toml

```diff
@@ -62,58 +62,27 @@ typish = '*'
 [tool.black]
 target-version = ['py38']
 
+[tool.pylint.'FORMAT']
+max-line-length = 120
```

Comment:
Ideally we should get this down to `100` which is the default

Reply:
im in favor of keeping it at 120.  It's the 21st century and we all have big screens